src/Makefile.in

```diff
@@ -30,10 +30,12 @@ RM = rm -f
 MKDIR = @MKDIR_P@
 TR = tr
 SED = @SED@
+FIND = @FIND@
 NULL = /dev/null
 SORT = @SORT@
 STRIP = @STRIP@
 CPPCHECK=cppcheck
```

Comment:
Shouldn't we also drop the `CPPCHECK` line?

Reply:
Yes, WIP.